packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -31,573 +31,28 @@ processors:
-      separator: ','
-      ignore_missing: true
-  - split:
-      field: http.request.headers.accept-languages
-      separator: ','
-      ignore_missing: true
-  # cs_header_names
-  - urldecode:
-      field: _tmp.cs_header_names
-      ignore_missing: true
-  - split:
-      description: List of cs header names. Truncated to 800 Bytes.
-      field: _tmp.cs_header_names
-      target_field: aws.cloudfront.cs_header_names
-      separator: '\n'
-      ignore_missing: true
-  - remove:
-      field: _tmp.cs_header_names
-      ignore_missing: true
-  # cs_header_count
-  - convert:
-      field: _tmp.cs_headers_count
-      description: Number of cs header names.
-      target_field: aws.cloudfront.cs_headers_count
-      type: integer
-      ignore_missing: true
-  - remove:
-      field: _tmp.cs_headers_count
-      ignore_missing: true
-  # asn
-  - rename:
-      field: _tmp.asn
-      target_field: aws.cloudfront.asn
-      ignore_missing: true
-
-
-  ### Cloudfront custom fields #####################################################################
-  - script:
-      lang: painless
-      description: This script adds some debug information on truncated headers.
-      source: |
-        def headers = ctx.http?.request?.headers;
-        def header_names = ctx.aws.cloudfront?.cs_header_names;
-        def headers_count = ctx.aws.cloudfront?.cs_headers_count;
-        def truncated_headers = 0;
-        def truncated_header_names = 0;
-
-        // Get tags
-        def tags = ctx.tags;
-        if (tags == null) {
-          tags = new ArrayList();
-        }
-
-        def tag_values = new ArrayList();
-
-        if (headers_count != null && header_names != null) {
-          if ( headers_count > header_names.length ) {
-            tags.add('cs_header_names_truncated')
-          }
-        }
-
-        if(headers_count != null && headers != null) {
-          if (headers_count > headers.size()) {
-            tags.add('cs_headers_truncated')
-          }
-        }
-
-        else if(header_names != null && headers != null) {
-          if (header_names.length > headers.size()) {
-            if (!tags.contains('cs_headers_truncated')) {
-              tags.add('cs_headers_truncated')
-            }
-          }
-        }
-
-        if (tags.length > 0) {
-          // Get tags
-          ctx.tags = tags;
-        }
-      tag: add_truncated_tags
-
-
-
-### Process additional fields ######################################################################
+        CHAR: .
+  - pipeline:
+      description: Process cloudfront realtime logs
+      if: ctx.first_char == '{'
+      name: '{{ IngestPipeline "pipeline_realtime_logs" }}'
+      tag: pipeline_realtime_logs
+      ignore_missing_pipeline: false
+  - pipeline:
+      description: Process cloudfront standard logs
+      if: ctx.first_char != '{'
+      name: '{{ IngestPipeline "pipeline_standard_logs" }}'
+      tag: pipeline_realtime_logs
```

Comment:
should probably be `pipeline_standard_logs`?

Reply:
Thanks